lib/pacemaker/pcmk_output.c

```diff
@@ -596,6 +867,8 @@ static pcmk__message_entry_t fmt_functions[] = {
     { "rscs-colocated-with-list", "xml", rscs_colocated_with_list_xml },
     { "stacks-constraints", "default", stacks_and_constraints },
     { "stacks-constraints", "xml", stacks_and_constraints_xml },
+    { "transition", "default", transition_default },
+    { "transition", "xml", transition_xml },
```

Comment:
There's already a rsc_action, created by the inject-rsc-action.  I'm thinking about renaming all the XML elements created by inject-whatever messages to inject_whatever for clarity, even though they're encapsulated within an element which would allow telling them apart.  Doing so might then make renaming this one to rsc_action be okay.

Reply:
Fixed.